aviatrix/resource_aviatrix_transit_external_device_conn.go

```diff
@@ -67,8 +67,8 @@ func resourceAviatrixTransitExternalDeviceConn() *schema.Resource {
 				Optional:     true,
 				Default:      "IPsec",
 				ForceNew:     true,
-				Description:  "Tunnel Protocol. Valid values: 'IPsec', 'GRE' or 'LAN'. Default value: 'IPsec'.",
-				ValidateFunc: validation.StringInSlice([]string{"IPsec", "GRE", "LAN"}, false),
+				Description:  "Tunnel Protocol. Valid values: 'IPsec', 'GRE' or 'LAN'. Default value: 'IPsec'. Case insensitive.",
```

Comment:
Should we add a diff suppress func so that we get no diffs if someone uses `tunnel_protocol = "lan"` for example?

Reply:
Added.